docs/advanced_topics/audit_log.rst

```diff
@@ -0,0 +1,32 @@
+.. _audit_log:
+
+Audit log
+=========
+
+Wagtail provides a mechanism to log actions performed on its objects. Common activities like page creation, update, deletion,
+locking and unlocking, revision scheduling, privacy changes are automatically logged at the model level.
+
+The Wagtail admin uses the action log entries to provide a site-wide and page specific history of changes. It uses a
+a registry of 'actions' that provide additional context for the logged action.
+
+The audit log-driven Page history replaces the revisions list page, but provide a filter for revision-specific entries.
+
+.. note:: The audit log does not replace revisions
+
+To provide additional logging for your site or package, invoke the :meth:`~LogEntryManger.log_action` manager method via ``LogEntry.objects.log_entry(object_instance, action)``
+and register a ``register_log_actions`` hook to describe your action (see :ref:`register_log_actions`).
+
+You can provide additional metadata by passing additional parameters:
+
+- `user`` - a user object.
```

Comment:
`user`` -> ``user`` (double backtick before user - Github markdown escapes this comment)

Reply:
Thank you @tomdyson. Sharp-eyed as always!